### ISSUE TITLE
[Effect Suggestion] Welcome to GTA Online

### ISSUE THREAD

**Issue by Misterannaberg** [gta-chaos-mod/ChaosModV#2466]
**Name of effect**
Welcome to GTA Online

**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [ ] Long
- [ ] Shot
- [X] No duration

**Effect Description**
You are getting killed by a Orbital cannon and you get a message from a barcode saying: 1-0 Lzzzzzzz cry

**Comment by pipecek** [comment]
Instant death is not at all interesting

**Comment by Misterannaberg** [comment]
But I think it would be funnier than something like suicide

**Comment by kolyaventuri** [comment]
Yeah this could be a variation upon "Suicide," however we won't be adding it as an additional effect.
